src/AppBundle/Helper/DeckValidationHelper.php

```diff
@@ -236,14 +254,8 @@ public function canIncludeCard($deck, $slot, $deck_options = []) {
 					}
 				}
 
-				if(isset($option->permanent) && $option->permanent) {
-					$permanent_valid = false;
-					//Not permanent and not Ravenous
-					if ($card->getPermanent() == $option->permanent && $card->getCode() != 89002) {
-						$permanent_valid = true;
-					} else {
-						continue;
-					}
+				if(isset($option->permanent) && $card->getPermanent() != $option->permanent) {

```

Comment:
This makes parallel Jenny work. I removed the check for the specific code 89002 in favor of the more permissive check for story assets above. Suzi was previously not able to have e.g. _Poisoned_ in her decks.